src/protocols/secure_channel/PASESession.cpp

```diff
@@ -242,8 +242,10 @@ CHIP_ERROR PASESession::SetupSpake2p(uint32_t pbkdf2IterCount, const uint8_t * s
         ReturnErrorOnFailure(PASESession::ComputePASEVerifier(mSetupPINCode, pbkdf2IterCount, salt, saltLen, mPASEVerifier));
```

Comment:
suggest to update above:
`context[32] --> context[kSHA256_Hash_Length]`

Reply:
Sure, will do.